docs/Tutorials/createWiFiConfig.md

```diff
@@ -11,42 +11,41 @@ After activation and configuration of an AMT device with a wireless profile, rem
 
 ## Find if a Device Support Wireless
 
-RPC can pull information to help determine if your current device supports wireless functionality.  See [Build RPC](../GetStarted/buildRPC.md) for steps on how to obtain the binary.
+RPC can pull information to help determine if your current device supports wireless functionality.  See [Build RPC](../Reference/RPC/buildRPC_Manual.md) for steps on how to obtain the binary.
```

Comment:
Use RPC's `amtinfo` feature to determine if your current device supports wireless functionality. For steps to obtain the RPC binary, see [Build RPC](../Reference/RPC/buildRPC_Manual.md).